src/functions/onCreateGame.ts

```diff
@@ -0,0 +1,27 @@
+import { Handler } from 'aws-lambda';
+import { createGame } from '../module/db';
+import { WebSocketAPIGatewayEvent, LambdaEventBody, LambdaResponse } from '../types';
+import { response } from '../utils/response';
+
+/**
+ * Handler for creating a game.
+ * @param {WebSocketAPIGatewayEvent} event Websocket API gateway event
+ */
+export const handler: Handler = async (event: WebSocketAPIGatewayEvent): Promise<LambdaResponse> => {
+  const body: LambdaEventBody = JSON.parse(event.body);
+  const { payload } = body;
+  const { games } = payload;
+
+  console.log('Adding game to the db table...');
+  console.log('Payload data', games);
+  console.log('Type of payload data', typeof games);
+  try {
+    if (games) await createGame(games);
+    else return response(400, 'Games attribute cannot be empty');
```

Comment:
I will be passing in gameName, gameType, and gameVersion when CREATE_GAME is called.
Also, I see that createGame in ../module/db takes in connections. Add the connection that created the game so it'll be an array of 1 element

Reply:
then how are we planning to add more users to a game? make another route, e.g. JOIN_GAME?

Reply:
@yummyblabla

Reply:
Yes, that's right

Reply:
will add that in the next PR then